### ISSUE TITLE
pip repository is no longer public

### ISSUE THREAD

**Issue by beauremus** [fermi-ad/acsys-python#65]
This was noticed by the datalogger-to-ml project, whose GitHub workflow build depends on having access to our pip repo.

We should figure out how to accommodate this.

**Comment by rneswold** [comment]
Our `--extra-url` wasn't working? Blocked by the firewall?

**Comment by beauremus** [comment]
That's correct. The build from GitHub using `pip install --extra-index-url https://www-bd.fnal.gov/pip3 acsys` produces
```
ERROR: Could not find a version that satisfies the requirement acsys (from versions: none)
ERROR: No matching distribution found for acsys
```
This context may be useful
https://github.com/fermi-controls/datalogger-to-ml/issues/38
https://github.com/fermi-controls/datalogger-to-ml/issues/39

**Comment by beauremus** [comment]
I just tested this from EAF and it works. I don't know that it's acceptable for this to be truly public.
